examples/all_in_one/grafana/provisioning/dashboards/SLO_Effective_Burn-rate.json

```diff
@@ -0,0 +1,757 @@
+{
```

Comment:
Please add description of this dashboard to [dashboards folder README](https://github.com/seznam/slo-exporter/tree/master/grafana_dashboards).

I see that this file is not currently up-to-date (some of the included dashboards are already missing) - description of the already merged dashboards will fixed in a separate PR.

Reply:
I am not sure if this is a good place. If tt should documented a prefer a description directly i grafana - as described here: https://grafana.com/docs/grafana/latest/best-practices/best-practices-for-creating-dashboards/

Reply:
I've added the description within the dashboard itself in https://github.com/seznam/slo-exporter/pull/32/commits/40ed092528c70358384c2bcae0b60c3e5b186e06